src/cmd-dev-overlay

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3 -u
+# Add content on top of a commit, handling SELinux labeling etc.
+# This is a more flexible (and correct) replacement for the CI flow developed
+# in https://github.com/smarterclayton/origin/blob/4de957b019aee56931b1a29af148cf64865a969b/images/os/Dockerfile
```

Comment:
Hmm, not sure I follow. Agreed it's more convenient, though in what way is it more correct?

Reply:
That code was e.g. picking up xattrs from the environment.  (Which there aren't any but in my local cosa usage I have ACLs set up).  And it was defaulting to the provided UID (which is 0 there, so again fine but...); we want to support running as non-uid-0 too.